alica_test_utility/src/TestContext.cpp

```diff
@@ -27,15 +27,7 @@ int TestContext::init(AlicaCreators& creatorCtx)
 
 int TestContext::init(AlicaCreators&& creatorCtx)
```

Comment:
I think `AlicaContext::init` retuns a bool? If so please change it here as well

Reply:
it is returning `int`